server/controllers/router.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
```

Comment:
Nice to see routes used. However probably redundant due to the small scale of the app and only one endpoint being on the route.